### DISCUSSION TITLE
在Windows 11下的問題

### DISCUSSION THREAD

**Discussion by han190** [rime/home#830] [General]
首先感謝提供這麽棒的開源輸入法框架！

在Windows 11的版本下安裝了小狼毫，發現右下角會出現兩個圖標。雖然不影響正常使用但是會有點亂……
![image](https://user-images.githubusercontent.com/13799118/124196365-bacf2380-da78-11eb-813c-7660483c154e.png)

另外，Windows 11原生的輸入法現在也換成了圓角矩形的設計，不知道小狼毫是否可以配置成類似的外觀？
![image](https://user-images.githubusercontent.com/13799118/124196466-f538c080-da78-11eb-8841-992a79d501ab.png)

**Comment by Sciroccogti** [comment]
我这里还出现了间断性不显示选词界面的情况，当然不排除 Win11 本身问题或者 NVIDIA 的驱动 bug。

**Comment by btstream** [comment]
我是win11 insider build 22489，出现开机无法启动输入法的问题

**Reply by Ausli** [reply]
相同问题

**Comment by LEOYoon-Tsaw** [comment]
出個win11版，不再支持老版win就好了

**Comment by rtransformation** [comment]
我也用的win11，我想你是除了Rime输入法以外还有别的输入法吧？把别的输入法都删去就好了。

**Comment by mianxiu** [comment]
要是能在window11下支持圆角加阴影就好了

**Comment by LearZhou** [comment]
非常奇怪，windows 11在管理員權限賬戶下輸入法沒問題，但在受限制賬戶下使用候選詞都是黑的，出不來。

這個很糟糕，因爲出於安全，最好在受限賬戶下使用。

**Reply by wang-yi-bit64** [reply]
解决了 杀掉后台重新运行下就好了

**Reply by 122414665** [reply]
圆角阴影

**Comment by iTomxy** [comment]
楼主这是在 win11 上用哪个版本啊？我装 0.14.3 有问题：

- 没有预设方案
- F4 和 Ctrl + ` 都呼不出菜单
- 自己用 /plum/ 装方案、写 default.custom.yaml 之后，点重新部署也无效

请问大家有遇到过这种问题吗？

---

（2023.6.15）小狼毫 `0.15.0`，win11 版本：`22H2（OS 内部版本 22621.1702）`。

之前用 0.14 和 0.9.30 的一个显著问题是：用 win + s 打开搜索之后，如果用小狼毫打字会卡死，再然后 win + s 索性呼不出系统搜索框了，但现在 0.15.0 好像没有这个问题，在 win + s 的搜索框中小狼毫中、英文都可正常使用。

还有之前在 win11 用 0.9.30 在系统菜单栏右击图标呼不出小狼毫菜单的问题，现在也可以了，如图：

<img width="264" alt="屏幕截图 2023-06-15 155545" src="https://github.com/rime/home/assets/44149820/be885d61-b76e-441a-8cec-29c661f295a3">

感觉 0.15.0 对 win11 的支持好一些。

**Reply by NSHSN** [reply]
我也在win11下遇到无法重新部署的问题，有人建议我退出算法服务后再重新运行，但是没有起作用，点重新部署出现“维护中”的弹窗

**Reply by wang-yi-bit64** [reply]
老哥解决没，我遇到这个问题 ，输入法都掉不起来了

**Comment by riverscn** [comment]
在菜单中选择“重新部署”试试

LearZhou ***@***.***>于2022年5月29日 周日04:53写道：

**Comment by Mct23** [comment]
![2023-06-23-1](https://github.com/rime/home/assets/137490258/84dc0724-bf38-45d4-9278-61a4453203c4)


自己做的仿win11微软拼音配色，供参考（字号等参数可能要重新调一下，我是2.5K 14寸屏）

```
  "style/display_tray_icon": false
  "style/horizontal": true #横排显示

  "style/font_point": 12 #字体大小
  "style/label_font_point": 12
  "style/inline_preedit": true # 嵌入式候选窗单行显示

  "style/layout/corner_radius": 14 #圆角幅度
  "style/layout/hilited_corner_radius": 12
  "style/layout/hilite_padding": 10

  "style/layout/hilite_spacing": 20 # 序号和候选字之间的间隔
  "style/layout/margin_x": 20 #候选字左右边距
  "style/layout/margin_y": 18 #候选字上下边距
  "style/layout/spacing": 40 #不知道干嘛的，在weasel下好像没有效果
  "style/layout/candidate_spacing": 40 # 候选字间隔
  "style/layout/round_corner": 0 #候选字背景色块圆角幅度

  "style/layout/border_width": 4

  "style/layout/shadow_radius": 10
  "style/layout/shadow_offset_x": 0
  "style/layout/shadow_offset_y": 6

    shadow_color: 0x26000000                           #整体阴影
    hilited_shadow_color: 0x26000000                            #编码背景块阴影颜色（inline predict模式无效）

    label_color: 0x5b5b5b #预选编号
    hilited_label_color: 0xD75A00 #已选择编号颜色 0x0067c0

    text_color: 0x191919 #已选择字 文字颜色
    back_color: 0xf9f9f9 #候选框 背景色
    border_color: 0xeeeeee #候选框 边框颜色

    candidate_text_color: 0x191919 #未候选字颜色 

    hilited_text_color: 0xF9f9f9 #已选择字右侧拼音 文字颜色
    hilited_back_color: 0xf9f9f9 #已选择字右侧拼音 背景色
    hilited_candidate_text_color: 0x191919 #候选字颜色
    hilited_candidate_back_color: 0xebebeb #候选字背景色
```

**Comment by Youxikong** [comment]
weasel.custom.yaml的patch下面加一句：
`"style/display_tray_icon": false` ，这样托盘就不会出现小狼毫的图标了。
